guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<blockquote>In the last layers of the backbone, we replace the traditional convolutional layers with Dilated convolution layers, which helps in increasing the receptive field. These Dilated convolution layers are placed in the last two blocks of the backbone. Hence the feature maps received at the end of the backbone contain richer features.&nbsp;Here is an illustration from&nbsp;<a href="https://towardsdatascience.com/review-dilated-convolution-semantic-segmentation-9d5a5bd768f5" target="_blank">this blog post</a>[2] showing what dilated convol.utions do</blockquote><p>In the last layers of resnet backbone, standard convolutional layers are replaced with Dilated convolution layers, which helps in increasing the receptive field. These Dilated convolution layers are placed in the last two blocks of the backbone. Hence the feature maps received at the end of the backbone contain richer features.&nbsp;Here is an illustration from&nbsp;<a href="https://towardsdatascience.com/review-dilated-convolution-semantic-segmentation-9d5a5bd768f5" target="_blank">this blog post</a> [2] showing what dilated convolutions do</p><ul><li>Hyperlink ""Dilated convolution" so users can read in detai <a href="https://arxiv.org/ftp/arxiv/papers/1709/1709.00179.pdf" target="_blank">https://arxiv.org/ftp/arxiv/papers/1709/1709.00179.pdf</a></li><li>It will give more clarity if the image showing standard convolution is also added alongside. For example at <a href="https://towardsdatascience.com/review-dilated-convolution-semantic-segmentation-9d5a5bd768f5" target="_blank">https://towardsdatascience.com/review-dilated-convolution-semantic-segmentation-9d5a5bd768f5</a></li><li>Explaining the concept of l=1 (standard convolution) and l&gt;1 (dilated convolution) with mathematical formula mentioned along.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Backbone can be anything so we should not specifically say "resnet backbone."</p><p>Rest, I have added standard convolution animation too.</p><p> </p>